VersionControlUI/CMakeLists.txt

```diff
@@ -8,8 +8,10 @@ add_library(VersionControlUI SHARED
 	src/VersionControlUIException.h
 	src/versioncontrolui_api.h
 	src/VersionControlUIPlugin.h
+	src/DiffManager.h
```

Comment:
In case you don't know, in git gui you can select entire files for commit, but you can also select just a few lines from a file and commit those. You should use this if a file contains multiple unrelated changes.